### PR TITLE
fix(content): make entire activity item row clickable

### LINKED ISSUE #316: Bigger Click Targets
This whole area needs to be clickable, not just the blue URL line. 

clicktarget

### PR DESCRIPTION
**Pull request by rlr**
r? @k88hudson 

Fixes #316

### CODE REVIEW

File: content-src/components/ActivityFeed/ActivityFeed.js [hunk 1]
```diff
@@ -46,16 +46,18 @@ const ActivityFeedItem = React.createClass({
...
-      <div className="feed-details">
-        <div className="feed-description">
-          <h4 className="feed-title" ref="title">{title}</h4>
-          <a className="feed-link" href={site.url} ref="link">{prettyUrl(site.url)}</a>
+      <a href={site.url} ref="link">
+        {icon}
+        <div className="feed-details">
+          <div className="feed-description">
+            <h4 className="feed-title" ref="title">{title}</h4>
+            <span className="feed-url" href={site.url} ref="url">{prettyUrl(site.url)}</span>
```
Comment (pdehaan) [inline content-src/components/ActivityFeed/ActivityFeed.js]: Can probably remove the `href` attr on the span,

Comment (rlr) [inline content-src/components/ActivityFeed/ActivityFeed.js]: ooops. good eye!

**Comment by k88hudson** [comment]
Nice!

**Comment by rlr** [comment]
gracias

**Pull request merged**